### ISSUE TITLE
[FR] read_feather without going through pandas

### ISSUE THREAD

**Issue by jangorecki** [modin-project/modin#278]
Currently we have
```py
read_feather(path, nthreads=1):
    warnings.warn("Defaulting to Pandas implementation", UserWarning)
    port_frame = pandas.read_feather(path)
    ...
```
Would be nice to have more native way of loading feather into modin. Filling this issue to be notified when that happens.

**Comment by devin-petersohn** [comment]
Thanks @jangorecki for posting this feature request!

I'm going to flag as good first issue because it should be relatively straightforward to do.

**Comment by devin-petersohn** [comment]
Resolved via #292

Performance numbers posted [here](https://github.com/modin-project/modin/pull/292#issuecomment-445915511)

**Comment by jangorecki** [comment]
Great, it was fast! thanks
